handsontable/src/plugins/touchScroll/touchScroll.js

```diff
@@ -20,6 +20,10 @@ export class TouchScroll extends BasePlugin {
     return PLUGIN_PRIORITY;
   }
 
+  static get ALWAYS_UPDATE() {
```

Comment:
I don't feel well about the `ALWAYS_UPDATE` option 🤔 &nbsp; sounds like a workaround. I can imagine that some of the plugins want to observe all options. Maybe, in that case, we can reuse the `CONFIG_KEYS` that would return an empty array for observing all settings and `false` for disabling observing at all? 

I just thinking aloud - Like `CONFIG_KEYS` sounds reasonable. We could support and maintain it in the future, even after implementing the "observability" to the MetaManager then `ALWAYS_UPDATE` ... I don't know :)

Reply:
That makes sense, but I'd need to change [the way I designed `CONFIG_KEYS` to work ](https://github.com/handsontable/handsontable/pull/9025#discussion_r766581366). One thing less to maintain, I guess.